modules/environment/nix.nix

```diff
@@ -27,6 +27,15 @@ in
   options = {
 
     nix = {
+      enable = mkOption {
```

Comment:
I don't want to support this option =)

Reply:
It's just there to make implementation from NixOS happy. There is also `{ config.nix.enable = true; }` below to prevent users to turn it off.

Maybe I should remove the `makeIf cfc.enable` stuff inside this `nix.nix`.